webapp/views/Guest/views/Login/Login.js

```diff
@@ -1,19 +1,19 @@
-import './loginView.scss'
+import './Login.scss'
 
 import React from 'react'
 import { useDispatch, useSelector } from 'react-redux'
 import { Link } from 'react-router-dom'
 
-import { useI18n, useFormObject } from '@webapp/components/hooks'
-import NotLoggedInView from '@webapp/guest/components/notLoggedInView'
-
-import { guestModules } from '@webapp/app/appModules'
+import NotLoggedIn from '@webapp/views/Guest/views/components/NotLoggedIn'
```

Comment:
this component can even go to webapp/components. what do u think?

Reply:
I had the same thought, so.... It means something